Sources/SwiftGRPCNIO/ServerCallContexts/UnaryResponseCallContext.swift

```diff
@@ -32,10 +32,7 @@ open class UnaryResponseCallContext<ResponseMessage: Message>: ServerCallContext
 ///
 /// We can use a protocol (instead of an abstract base class) here because removing the generic `responsePromise` field
 /// lets us avoid associated-type requirements on the protol.
```

Comment:
Not related to your change, but just noticed a typo of "protol" -> "protocol" if you want to fix it while you're here

Reply:
Done, thanks!